WikipediaUnitTests/Utilities/XCTestCase+MWKFixtures.h

```diff
@@ -36,7 +36,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (MWKArticle*)articleWithMobileViewJSONFixture:(NSString*)fixtureName
                                       withTitle:(id)titleOrString
-                            insertIntoDataStore:(MWKDataStore*)dataStore;
+                                      dataStore:(MWKDataStore*)dataStore;
```

Comment:
Per Brian's suggestion. "Insert" wasn't entirely accurate because it doesn't call "[article save]".